### DISCUSSION TITLE
INVALID HEADER   Missing required header field: "Date"

### DISCUSSION THREAD

**Discussion by IvRRimum** [plausible/analytics#1045] [Self-Hosted Support]
### Precheck

Please note that this tracker is only for bugs. Do not use the issue tracker for help, support or feature requests. 

[Our docs](https://plausible.io/docs) are a great place for most answers, but if you can’t find your answer there, you can [contact us](https://plausible.io/contact). 

Have a feature request? Please search the ideas [on our forum](https://github.com/plausible/analytics/discussions/categories/feature-requests) to make sure that the feature has not yet been requested. If you cannot find what you had in mind, please [submit your feature request here](https://github.com/plausible/analytics/discussions/new). 

Have an issue with your self-hosted install? You can ask in [our self-hosted forum](https://github.com/plausible/analytics/discussions/categories/self-hosted-support). Thanks!

## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.
YES

## Bug report
**Describe the bug**

My SMTP is setup and it works for other services.
Why trying to reset password, i get this email:

```
The message WAS NOT relayed to:
 <info@my-domain.com>:
  554 5.6.0 Bounce, id=00194-12 - BAD HEADER

This nondelivery report was generated by the program amavisd-new at host
mail.my-domain.com. Our internal reference code for your message is
00194-12/uJ0lBkaam0J9

INVALID HEADER

 Missing required header field: "Date"
```

**Expected behavior**
I expect to receive an password recovery email.

**Screenshots (If applicable)**

**Environment (If applicable):**
 - OS: Ubuntu
 - Browser: No.
 - Version
 This commit is my newest: 3adc80359b5247ed06ede99a327579da6dd8999c

**Comment by IvRRimum** [comment]
How is this not a bug?

**Comment by celevra** [comment]
this is clearly a bug, it is not only the date field also Message ID, Mime Type Text as Alternative, the Subject is in Base64 but its not needed, and so on

<img width="230" alt="image" src="https://user-images.githubusercontent.com/232437/146553578-731f68b1-3154-4274-bce0-2e5eb0596739.png">

**Comment by kymppi** [comment]
This is still happening on currently the [latest docker image](https://hub.docker.com/layers/plausible/analytics/latest/images/sha256-3fbdcfac56ffa79e778e9e4bc1ca8c4dca0d7deb330ed5694f59eaa1125cc0a8?context=explore)
